src/Microsoft.ML.TimeSeries/SsaChangePointDetector.cs

```diff
@@ -141,5 +184,99 @@ public override void Save(ModelSaveContext ctx)
 
             base.Save(ctx);
         }
+
+        // Factory method for SignatureLoadRowMapper.
+        private static IRowMapper Create(IHostEnvironment env, ModelLoadContext ctx, ISchema inputSchema)
+            => Create(env, ctx).MakeRowMapper(inputSchema);
+    }
+
+    /// <summary>
+    /// Estimator for <see cref="SsaChangePointDetector"/>
+    /// </summary>
+    public sealed class SsaChangePointEstimator : IEstimator<SsaChangePointDetector>
+    {
+        private readonly IHost _host;
+        private readonly string _inputColumnName;
+        private readonly string _outputColumnName;
+        private readonly int _confidence;
+        private readonly int _changeHistoryLength;
+        private readonly int _trainingWindowSize;
+        private readonly int _seasonalityWindowSize;
+        private readonly MartingaleType _martingale;
+        private readonly double _powerMartingaleEpsilon;
+        private readonly ErrorFunctionUtils.ErrorFunction _errorFunction;
+
+        /// <summary>
+        /// Convinence constructor
+        /// </summary>
+        /// <param name="env">Host Environment.</param>
+        /// <param name="name">The name of the new column.</param>
+        /// <param name="source">Name of the input column.</param>
+        /// <param name="confidence">The confidence for change point detection in the range [0, 100].</param>
+        /// <param name="trainingWindowSize">The change history length.</param>
+        /// <param name="changeHistoryLength">The change history length.</param>
+        /// <param name="seasonalityWindowSize">The change history length.</param>
+        /// <param name="errorFunction">The function used to compute the error between the expected and the observed value.</param>
+        /// <param name="martingale">The martingale used for scoring.</param>
+        /// <param name="eps">The epsilon parameter for the Power martingale.</param>
+        public SsaChangePointEstimator(IHostEnvironment env, string name, string source,
+            int confidence, int changeHistoryLength, int trainingWindowSize, int seasonalityWindowSize,
+            ErrorFunctionUtils.ErrorFunction errorFunction = ErrorFunctionUtils.ErrorFunction.SignedDifference,
+            MartingaleType martingale = MartingaleType.Power, double eps = 0.1)
+        {
```

Comment:
> { [](start = 7, length = 2)

Why don't we create an instance of arguments here, instead of duplicating all fields?